equinix-openapi-metal/src/main/java/com/equinix/openapi/metal/v1/model/Href.java

```diff
@@ -145,9 +144,9 @@ public static void validateJsonElement(JsonElement jsonElement) throws IOExcepti
         }
       }
 
-      Set<Entry<String, JsonElement>> entries = jsonElement.getAsJsonObject().entrySet();
+      Set<Map.Entry<String, JsonElement>> entries = jsonElement.getAsJsonObject().entrySet();
```

Comment:
This is an upstream bugfix to avoid errors when an OpenAPI spec defines an `Entry` schema.  When `java.util.Map.Entry` is imported explicitly, it is referenced as `Entry`, which could conflict with a generated class name; when the explicit import is removed, this class is referred to as `Map.Entry` instead.